backend/src/main/java/com/happy/friendogly/playground/service/PlaygroundCommandService.java

```diff
@@ -21,24 +23,31 @@
 import com.happy.friendogly.playground.repository.PlaygroundRepository;
 import java.time.LocalDateTime;
 import java.util.List;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @Transactional
 public class PlaygroundCommandService {
 
+    public static final int OUTSIDE_MEMBER_KEEP_TIME = 2;
+    public static final String EVERY_30_MINUTE = "0 0/30 * * * *";
+
```

Comment:
상수화가 깔끔하네요~~~ 단위만 추가해줍시다

```suggestion
    public static final int OUTSIDE_MEMBER_KEEP_TIME_AS_HOURS = 2;
    public static final String EVERY_30_MINUTE = "0 0/30 * * * *";

```